src/types.rs

```diff
@@ -82,6 +82,8 @@ pub struct LanguageServerConfig {
     pub settings_section: Option<String>,
     pub settings: Option<Value>,
     pub offset_encoding: Option<OffsetEncoding>,
+    #[serde(default)]
+    pub symbols_mapping: HashMap<String, String>,
```

Comment:
```suggestion
    pub symbols_mapping: HashMap<SymbolKind, String>,
```
Would be nice, but not sure how.

Reply:
we'd need to write an

	impl<'de> Deserialize<'de> for SymbolKind

should be fairly simple but I think we might wanna try a different UI